projects/common-components/src/lib/pipes/sort/sort.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { Talk } from "@jsmu/core";
+
+@Pipe({
+	name: "sort",
+	pure: false
```

Comment:
using dirty pipe is bad practice, can affect performance

Reply:
done